### PR TITLE
fix: Move import statements to the top of extracted index.css file.

### PR DESCRIPTION
**Pull request by liamcho**
Fixes: [CLNP-3084](https://sendbird.atlassian.net/browse/CLNP-3084)

### Changelogs
- Fixed a bug where import statements are not located at the top of the extracted index.css file

[CLNP-3084]: https://sendbird.atlassian.net/browse/CLNP-3084?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: rollup.config.mjs [hunk 1]
```diff
@@ -14,6 +14,7 @@ import ts2 from "rollup-plugin-typescript2"
 // config from package.json
 import pkg from "./package.json" assert {type: "json"};
 import inputs from "./rollup.module-exports.mjs";
+import {readFileSync, writeFileSync} from 'fs';
```
Comment (bang9) [inline rollup.config.mjs:17]:
```suggestion
import { readFileSync, writeFileSync } from 'fs';
```

File: rollup.config.mjs [hunk 2]
```diff
@@ -61,6 +62,41 @@ export default {
       extract: "dist/index.css",
       extensions: [".sass", ".scss", ".css"],
     }),
+    {
```
Comment (bang9) [inline rollup.config.mjs:65]: When you compile Sass, it generates a CSS map (sourcemap) file that helps to track the positions of CSS and original Sass files. However, if the order of text in the CSS file changes, the sourcemap may no longer be valid.

Regarding whether there will be any impact on this issue (whether it's safe to remove it), personally, I think setting sourcemap: false shouldn't cause significant problems.


```js
postcss({
  // ...
  sourceMap: false,
})
```

Comment (bang9) [reply rollup.config.mjs:65]: Hmm.. let's remove it

**Comment by bang9** [comment]
could you update the title ? `bug-fix` -> `fix`
reference: https://www.conventionalcommits.org/en/v1.0.0/

File: rollup.config.mjs (hunk 2 above)
Comment (liamcho) [reply rollup.config.mjs:65]: interesting. Ill look into this maybe next week due to other priorities

Comment (liamcho) [reply rollup.config.mjs:65]: @bang9 I asked chatGPT and below is answer

> If you add a custom plugin like 'move-imports-to-top', it won't automatically handle sourcemaps for you. Sourcemaps are typically handled by the plugins provided by Rollup or other build tools.

It also suggested me to handle mapping manually using
`import { SourceMapConsumer, SourceMapGenerator } from 'source-map';`

Do you think we should try this? or just set sourceMap: false?

Comment (bang9) [reply rollup.config.mjs:65]: `sourceMap: false` let's go~

**Review by bang9 — APPROVED** [review]

**Pull request merged**